### PR TITLE
S2task1 - ugly form and image links working

### PR DESCRIPTION
**Pull request by nickclifford904**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
-$query = $db->prepare('SELECT * FROM `chaircollection`;');
-$query->setFetchMode(PDO::FETCH_ASSOC);
+$db = connectdb();
+$query = $db->prepare('SELECT `chairname`, `designer`, `designyear`, `imagelink` FROM `chaircollection`;');
 $query->execute();
+?>
+    <h2>Please add more chairs to the collection!</h2>
+    <div class="chairinputform">
+    <form method="post" action="functions.php">
+        Chair name : <input type="text" name="chairname" size="50" placeholder="Enter the name of the chair" /><br>
```
Comment (imypullan) [inline index.php]: maybe put these in `div` instead?

File: index.php [hunk 2]
```diff
@@ -1,17 +1,35 @@
...
+?>
+    <h2>Please add more chairs to the collection!</h2>
+    <div class="chairinputform">
+    <form method="post" action="index.php">
+        <div>Chair name : <input type="text" name="chairname" size="50" placeholder="Enter the name of the chair" /></div>
+        <div>Designer's name : <input type="text" name="designer" size="50" placeholder="Enter the name of the designer" /></div>
+        <div>Year : <input type="number" name="designyear" size="40" placeholder="Enter the year of the design" /></div>
+        <div>Wikimedia Image Link : <input type="text" name="imagelink" size="40" placeholder="Copy and paste Wikimedia Image Link" /></div>
+        <input name="submit" type="submit" value="Submit" />
+    </form></div>
```
Comment (ashleycoles) [inline index.php:26]: Just double check your indenting here, the form should be one level further to show its in the div

File: index.php [hunk 3]
```diff
@@ -1,17 +1,35 @@
...
+
 
-    echo '<ul>';
-    foreach ($query as $chair) {
-        echo '<li>The ' . $chair['chairname'] . ' was designed by ' . $chair['designer'] . ' in ' . $chair['designyear'] . '.</li>';
-    }
-    echo '</ul>';
 
+$result = displayChairs($query);
+echo $result;
```
Comment (ashleycoles) [inline index.php:35]: Make sure this is happening inside of your <body> tag - all visible content must be contained within the body

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Just a few minor changes, really nice work!

**Pull request merged**